coupledmodeldriver/platforms.py

```diff
@@ -10,6 +10,14 @@ class Platform(Enum):
         'slurm_account': 'coastal',
         'default_partition': None,
     }
+    ORION = {
+        'source_filename': '/work/noaa/nosofs/share/repositories/ADC-WW3-NWM-NEMS/modulefiles/envmodules_intel.hera',
+        'processors_per_node': 40,
+        'launcher': 'srun',
+        'uses_slurm': True,
+        'slurm_account': 'nosofs',
+        'default_partition': None,
+    }
```

Comment:
@saeed-moghimi-noaa are these values (processors per node and slurm account) correct? Also, does Orion require a partition like TACC?